src/mp4file.cpp

```diff
@@ -745,7 +745,8 @@ void MP4File::FindIntegerProperty(const char* name,
     if ( !FindProperty( name, ppProperty, pIndex ) ) {
         ostringstream msg;
         msg << "no such property - " << name;
-        throw new Exception( msg.str(), __FILE__, __LINE__, __FUNCTION__ );
+        log.errorf( "MP4File::FindIntegerProperty - %s", msg.str().c_str() );
+        return;
```

Comment:
FWIW, I think throwing was the original behavior here.

Reply:
yeah this revert (Revert A) gets undone in the next revert (Revert B), it's a bit confusing here 😁 but if I only reverted that next commit (Revert B) there would have been a conflict with this commit